test/NCI.OCPL.Api.DrugDictionary.Tests/Tests/Controllers/DrugsControllerTests.GetAll.cs

```diff
@@ -122,6 +124,43 @@ public async void GetAll_InvalidFrom()
             );
         }
 
+        /// <summary>
+        /// Verify the controller gracefully handles errors in the service layer.
+        /// </summary>
+        [Theory]
+        [InlineData(typeof(APIInternalException))]
+        //[InlineData(typeof(ArgumentNullException))]
```

Comment:
Pretty sure you meant to uncomment that, right? 😉

Reply:
Aw, nuts.